Sources/swift-format/main.swift

```diff
@@ -50,6 +50,87 @@ fileprivate func main(_ arguments: [String]) -> Int32 {
   }
 }
 
+/// Iterator for looping over lists of files and directories. Directories are automatically
```

Comment:
Can you move this type to a `FileIterator.swift` file in the `swift-format` module?

Reply:
Done.